sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -557,3 +559,63 @@ def upsert_entity(  # pylint:disable=R1710
             return _trim_service_metadata(metadata)
         except HttpResponseError as error:
             _process_table_error(error)
+
+    def create_batch(
+        self,
+        **kwargs # type: Dict[str, Any]
+    ):
+        # type: (...) -> TableBatchOperations
+        """Create a Batching object from a Table Client
+
+        :return: Object containing requests and responses
+        :rtype: TableBatchOperations
+        :raises: ~azure.core.exceptions.HttpResponseError
```

Comment:
`create_batch` doesn't make any API calls does it? In which case it wont be raising an HTTP error